### PR TITLE
Add missing clone method to Course class

### PR DESCRIPTION
**Pull request by henning-gerhardt**
According to CodeQL the Course class is missing a `clone` method to extend the `ArrayList` base class complete.

### CODE REVIEW

**Review by solth — CHANGES_REQUESTED** [review]
Please rebase against current master

File: Kitodo/src/test/resources/newspaper-course.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (henning-gerhardt) [reply Kitodo/src/test/resources/newspaper-course.xml:1]: License header added.

File: Kitodo/src/test/java/org/kitodo/production/model/bibliography/course/CourseTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import java.nio.file.Files;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.junit.jupiter.api.Test;
+import org.kitodo.production.helper.XMLUtils;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+class CourseTest {
```
[the remark this answers is not in the document]
Comment (henning-gerhardt) [reply Kitodo/src/test/java/org/kitodo/production/model/bibliography/course/CourseTest.java]: Made class and method public.

**Comment by henning-gerhardt** [comment]
PR is rebased.

**Review by solth — APPROVED** [review]

**Pull request merged**